extensions/ql-vscode/src/data-extensions-editor/auto-model-codeml-queries.ts

```diff
@@ -284,3 +211,29 @@ export async function generateCandidateFilterPack(
 
   return packDir;
 }
+
+async function interpretAutomodelResults(
+  cliServer: CodeQLCliServer,
+  completedQuery: CoreCompletedQuery,
+  metadata: QueryMetadata,
+  sourceInfo: SourceInfo | undefined,
+): Promise<Sarif.Log> {
+  const interpretedResultsPath = join(
+    completedQuery.outputDir.querySaveDir,
+    "results.sarif",
+  );
+
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars -- We only need the actual SARIF data, not the extra fields added by SarifInterpretationData
+  const { t, sortState, ...sarif } = await interpretResultsSarif(
```

Comment:
Minor: Looks like you can do this:

```suggestion
  const { ...sarif } = await interpretResultsSarif(
```